src/functionalTest/resources/acceptance_criteria/rdm-13170/ac.F-001/AC-1.feature

```diff
@@ -0,0 +1,21 @@
+#====================================
```

Comment:
Can you double-check this file please - I'm not sure what it's purpose is? Talking about invalid scheme, returning a 400, but then the expected is to return a 200. I don't recognise this directory that it's in either.